dist/redhat/scylla.spec

```diff
@@ -141,7 +141,9 @@ rm -rf $RPM_BUILD_ROOT
 %ghost /etc/systemd/system/scylla-server.service.d/capabilities.conf
 %ghost /etc/systemd/system/scylla-server.service.d/mounts.conf
 /etc/systemd/system/scylla-server.service.d/dependencies.conf
+%ghost %config /etc/systemd/system/var-lib-systemd-coredump.mount
 %ghost /etc/systemd/system/scylla-cpupower.service
+%ghost %config /etc/systemd/system/var-lib-scylla.mount
```

Comment:
I thought we'll need two separate lines, good to see it works in one line (it means the rpm maintainers considered this case)

Reply:
@avikivity it just worked with one line, and also rpmlint does not causes error with this.
I could find one line examples on the web:
https://git.sailfishos.org/mer-core/rpm/blob/master/rpm/rpm.spec#L216

Also, it seems maintainers are thinking it's expected behavior:
https://bugzilla.redhat.com/show_bug.cgi?id=1516938
https://github.com/rpm-software-management/rpm/commit/233e4573d1d21a36d06874fea6c594428033802e